1.md

```diff
@@ -0,0 +1,3 @@
+Как незарегистрируемый пользователь,  
+я хочу иметь возможность зарегистрироваться с использованием уникального логина, 
+чтобы другие пользователи могли найти и идентифицировать меня.
```

Comment:
I'm not sure about this motivation part. Can you think of any other reason to have a unique account?

Reply:
I've added another motivation